lib/active_model_serializers/adapter.rb

```diff
@@ -1,7 +1,7 @@
 module ActiveModelSerializers
   module Adapter
     UnknownAdapterError = Class.new(ArgumentError)
-    ADAPTER_MAP = {}
+    ADAPTER_MAP = {} # rubocop:disable Style/MutableConstant
```

Comment:
Rubocop changed this to:
```ruby
ADAPTER_MAP = {}.freeze
```

which prevent us to modify the hash at [line 52](https://github.com/groyoh/active_model_serializers/blob/update_rubocop/lib/active_model_serializers/adapter.rb#L52) and thus breaks the gem/tests.